Mathlib/Topology/Homotopy/Lifting.lean

```diff
@@ -220,9 +220,13 @@ theorem exists_path_lifts : ∃ Γ : C(I,E), p ∘ Γ = γ ∧ Γ 0 = e := by
     exact ⟨⟨Γ, continuous_iff_continuousOn_univ.mpr
       (by convert cont; rw [eq_comm, Set.eq_univ_iff_forall]; exact fun t ↦ ⟨bot_le, le_top⟩)⟩,
       funext fun _ ↦ eqOn ⟨bot_le, le_top⟩, Γ_0⟩
-  refine Nat.rec ⟨fun _ ↦ e, continuous_const.continuousOn, fun t ht ↦ ?_, rfl⟩
-    fun n ⟨Γ, cont, eqOn, Γ_0⟩ ↦ ?_
-  · rw [t_0, Set.Icc_self] at ht; cases ht; exact γ_0.symm
+  intro n
```

Comment:
This code is more structured but longer: I personally think this is fine, but flagging this for a second opinion.

Reply:
Yeah, I wouldn't necessarily mind dropping this change, but I think using `Nat.rec` manually in a proof tends to be golfing too much.